Installer.iss

```diff
@@ -96,4 +96,4 @@ Filename: "{app}\{#MyAppExeName}"; Description: "{cm:LaunchProgram,{#StringChang
 UseRelativePaths=True
 
 [Messages]
-SelectDirBrowseLabel=To continue, click Next.  If you have a custom VoiceAttack installation location, or would like to put the EDDI files in a different location, click Browse.
+SelectDirBrowseLabel=To continue, click Next. If this is not your VoiceAttack installation location, or you would like to put the EDDI files in a different location, click Browse.
```

Comment:
Proposed re-write. TMI?
```
To continue, click Next. If this is not your VoiceAttack installation location, or if you would like to install EDDI in a different location, click Browse.
Note that EDDI must be installed in the "\Apps" subdirectory of VoiceAttack to function as a VoiceAttack plugin (e.g. "C:\Program Files (x86)\VoiceAttack}\Apps\EDDI").
```

Reply:
Unless you feel very strongly about it, I would prefer to keep the text as brief as possible. In my experience, beyond a certain length pretty much everyone just skips reading it altogether.

It's already rather longer than I'd like.